### PR TITLE
Xhb Adapter: adding alias support

### PR DESCRIPTION
**Pull request by daniel-hoffmann**
## Type of change
- [x ] Other

## Description of change
Changed Xhb Adapter to allow aliasing

- contact email of the adapter’s maintainer: daniel.hoffmann@xaxis.com

Let me know if any changes need to be made.

### CODE REVIEW

**Comment by daniel-hoffmann** [comment]
@jaiminpanchal27 Can you please check if the most recent commit is okay? Thanks! :)

File: modules/xhbBidAdapter.js [hunk 1]
```diff
@@ -1,44 +1,42 @@
...
+
+  baseAdapter.callBids = function (params) {
+    const anArr = params.bids;
+    for (let i = 0; i < anArr.length; i++) {
+      let bidRequest = anArr[i];
+      let callbackId = bidRequest.bidId;
+      loadScript(buildJPTCall(bidRequest, callbackId));
+    }
   };
-  bidmanager.registerDefaultBidderSetting('xhb', _defaultBidderSettings);
```
Comment (mkendall07) [inline modules/xhbBidAdapter.js]: Was this drop intentional?

File: modules/xhbBidAdapter.js [hunk 2]
```diff
@@ -106,24 +86,25 @@ function XhbAdapter() {
...
       let bid = [];
       if (jptResponseObj.result && jptResponseObj.result.ad && jptResponseObj.result.ad !== '') {
         responseCPM = 0.00;
 
-          // store bid response
-          // bid status is good (indicating 1)
+        // store bid response
+        // bid status is good (indicating 1)
         let adId = jptResponseObj.result.creative_id;
-        bid = bidfactory.createBid(CONSTANTS.STATUS.GOOD, bidObj);
```
Comment (mkendall07) [inline modules/xhbBidAdapter.js]: Please keep the constant value here

File: modules/xhbBidAdapter.js [hunk 3]
```diff
@@ -134,30 +115,37 @@ function XhbAdapter() {
 
         bidmanager.addBidResponse(placementCode, bid);
       } else {
-          // no response data
-          // indicate that there is no bid for this placement
-        bid = bidfactory.createBid(2);
+        // no response data
+        // indicate that there is no bid for this placement
+        bid = bidfactory.createBid(2, bidObj);
```
Comment (mkendall07) [inline modules/xhbBidAdapter.js]: use constant value

**Comment by mkendall07** [comment]
@daniel-hoffmann 
had a question and few minor changes and can merge.

File: modules/xhbBidAdapter.js (hunk 2 above)
Comment (daniel-hoffmann) [inline modules/xhbBidAdapter.js]: Done

File: modules/xhbBidAdapter.js (hunk 1 above)
Comment (daniel-hoffmann) [inline modules/xhbBidAdapter.js]: It was originally, yes - but on second though, it probably is better to keep it in for the time being, so I added it in again.

**Comment by mkendall07** [comment]
LGTM.

**Pull request merged**